static/app/views/insights/cache/components/charts/hitMissChart.tsx

```diff
@@ -1,41 +1,21 @@
-import {useTheme} from '@emotion/react';
-
-import type {Series} from 'sentry/types/echarts';
-import {formatPercentage} from 'sentry/utils/number/formatPercentage';
-import {CHART_HEIGHT} from 'sentry/views/insights/cache/settings';
-import {AVG_COLOR} from 'sentry/views/insights/colors';
-import Chart, {ChartType} from 'sentry/views/insights/common/components/chart';
-import ChartPanel from 'sentry/views/insights/common/components/chartPanel';
+import {InsightsLineChartWidget} from 'sentry/views/insights/common/components/insightsLineChartWidget';
+import type {DiscoverSeries} from 'sentry/views/insights/common/queries/useDiscoverSeries';
 import {DataTitles} from 'sentry/views/insights/common/views/spans/types';
 
 type Props = {
   isLoading: boolean;
-  series: Series;
+  series: DiscoverSeries;
   error?: Error | null;
 };
 
 export function CacheHitMissChart({series, isLoading, error}: Props) {
-  const theme = useTheme();
   return (
-    <ChartPanel title={DataTitles[`cache_miss_rate()`]}>
-      <Chart
-        height={CHART_HEIGHT}
-        grid={{
-          left: '4px',
-          right: '0',
-          top: '8px',
-          bottom: '0',
-        }}
-        data={[series]}
-        loading={isLoading}
-        error={error}
-        chartColors={[AVG_COLOR(theme)]}
-        type={ChartType.LINE}
-        aggregateOutputFormat="percentage"
-        tooltipFormatterOptions={{
-          valueFormatter: value => formatPercentage(value),
-        }}
-      />
-    </ChartPanel>
+    <InsightsLineChartWidget
+      title={DataTitles[`cache_miss_rate()`]}
+      series={[series]}
+      showLegend="never"
+      isLoading={isLoading}
+      error={error ?? null}
```

Comment:
perhaps we allow the widget to also except `undefined`?

Reply:
Definitely maybe, I'll think about it! I'll add that to my list of improvements 👍🏻